quickstart/faqs.md

```diff
@@ -1,62 +1,70 @@
+This OpenEBS Mayastor documentation contains sections which are focused on initial, 'quickstart' deployment scenarios, including the correct configuration of underlying hardware and software, and of Mayastor features such as "Storage Nodes" \(MSNs\) and "Disk Pools" \(MSPs\).  Information describing tuning for the optimisation of performance is also provided.
 
 * [Quickstart Guide](https://mayastor.gitbook.io/introduction/quickstart/configure-mayastor)
 * [Performance Tips](https://mayastor.gitbook.io/introduction/quickstart/performance-tips)
 
-MayaData provides resources which can automatate both these types of configuration tasks.  These include the ["automation playground" project](https://github.com/mayadata-io/deployment-automation-playground/tree/main/demo-playground), which uses Terraform and Ansible, is pluggable and extensible, and is able to deploy both FIO based benchmarking and other workloads for initial "first sighting" deployments.
+MayaData has authored resources which can automatate both these types of configuration tasks.  These include the open source ["automation playground" project](https://github.com/mayadata-io/deployment-automation-playground/tree/main/demo-playground), which uses Terraform and Ansible, is pluggable and extensible, and is able to deploy both FIO based benchmarking and other workloads for initial "first sighting" deployments.  Contributions, extensions and suggestions are welcome.
 
 ## What is the basis for its performance claims?  Has it been benchmarked?
-Mayastor has been built to leverage the performance potential of contemporary, high-end,  solid state storage devices as a foremost design consideration.  For this reason, the I/O path is predicated on NVMe, a transport which is both highly CPU efficient and which demonstrates highly linear resource scaling.  The data path runs entirely within user space, also contributing efficiency gains as syscalls are avoided, and is both interrupt and lock free.
+Mayastor has been built to deliver as close as possible to local disk performance with a specific design goal of less than 10% overhead versus the performance capabilities of underlying devices including always improving high performance SSDs; while this may sound like common sense, <10% overhead is much lower overhead than traditional storage systems and storage software which often impart 40-50% or greater overhead. For this reason, the Mayastor I/O path is predicated on NVMe, a transport which is both highly CPU efficient and which demonstrates highly linear resource scaling. The data path runs entirely within user space, also contributing efficiency gains as syscalls are avoided, and is both interrupt and lock free; in other words, Mayastor like many high performance networking systems bypasses the kernel in order to deliver higher performance.

```

Comment:
Can we also provide the core data structure(s) that this storage system relies on? For example, is it BTree or LSM or something else. Are there any reference papers that has inspired the development of this storage system?

Reply:
I have issues with the factual correctness of some assertions:
 - including always improving high performance SSDs;
 - Mayastor like many high performance networking systems bypasses the kernel

In the first case, that's just not true, as an absolute statement.  We know already that there is some overhead, and for write intensive workloads, it is significant.  It certainly doesn't "always" improve.

Mayastor might bypass the kernel but the I/O path in general does not, not in its entirety.  It's still dependent on the kernel initiator of the worker node which is mounting the volume.

Reply:
On structures, this question is already asked (and answered) specifically in another question - "What is the on-disk format"